supabase/migrations/20250117160000_error_logs_table.sql

```diff
@@ -0,0 +1,65 @@
+-- Create error_logs table for logging errors from chat API
+-- This table stores structured error data for monitoring and analysis
+create table if not exists "public"."error_logs" (
+    "id" uuid not null default gen_random_uuid(),
+    "created_at" timestamp with time zone not null default now(),
+    "raw_message" text,
+    "telegram_message_id" bigint,
+    "user_email" text,
+    "room_id" uuid,
+    "error_timestamp" timestamp with time zone,
```

Comment:
make this of type TIMESTAMPZ (reference neighboring schema files)
- might already be the same with different notation than I expect.